.github/ISSUE_TEMPLATE/1-bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Read before posting:
+
+- Questions should be posted to https://community.grafana.com
+- Search/filter through opened and closed issues for existing similar bug reports: https://github.com/grafana/grafana/issues?utf8=%E2%9C%93&q=is%3Aissue
+- Checkout FAQ: https://community.grafana.com/c/howto/faq
+- Checkout How to troubleshoot metric query issues: https://community.grafana.com/t/how-to-troubleshoot-metric-query-issues/50
+- Checkout Using Grafana’s Query Inspector to troubleshoot issues: https://community.grafana.com/t/using-grafanas-query-inspector-to-troubleshoot-issues/2630
+
+Please answer and include relevant information below:
+
+### What Grafana version are you using?
+### What datasource are you using?
+### What OS are you running grafana on?
+### What did you do?
+### What was the expected result?
+### What happened instead?
+### If related to metric query / data viz
+### Include raw network request & response
+See Using Grafana’s Query Inspector to troubleshoot issues and/or How to troubleshoot metric query issues above.
```

Comment:
Should be rewritten without links in mind: "See our docs on the Query Inspector how to debug issues with metric queries." Perhaps a shortlink might help.